cmd/testdata/TestCreateCluster_splitkeys_output.golden

```diff
@@ -9,10 +9,10 @@ Created charon cluster:
  --split-existing-keys=true
 
 charon/
-├─ cluster-lock.json	Cluster lock defines the cluster lock file which is signed by all nodes
-├─ deposit-data.json	Deposit data file is used to activate a Distributed Validator on DV Launchpad
-├─ node[0-3]/		Directory for each node
-│  ├─ charon-enr-private-key		Charon networking private key for node authentication
+├─ node[0-3]/			Directory for each node
```

Comment:
best not hardcode 3

Reply:
It's not hardcoded: https://github.com/ObolNetwork/charon/blob/main/cmd/createcluster.go#L445

It's always four since the create_cluster testcases use 4 as num_nodes